packages/core/src/icon/icon.stories.mdx

```diff
@@ -24,6 +24,17 @@ ClarityIcons.addIcons(userIcon);
 <cds-icon shape="user"></cds-icon>
 ```
 
+In JavaScript, import the icon name from `@clr/core/icon` instead. This also provides type safety when used in TypeScript:
```

Comment:
nit: let's clarify that this is JSX/React specific syntax so users are not confused on how to use the component. Something like this maybe,

```
If using JSX or TSX, import the icon name from `@clr/core/icon` and include it in the `shape` prop to improve type safety.
```

Reply:
Done!